### DISCUSSION TITLE
remote: error: File node_modules/.cache/default-development/11.pack is 110.44 MB; this exceeds GitHub's file size limit of 100.00 MB

### DISCUSSION THREAD

**Discussion by DMitu** [community/community#51464] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

remote: Resolving deltas: 100% (18092/18092), done.
remote: error: Trace: 070b69b6d4e44ee3df0e8070f13a09be78ed766bc07d90da4edbf934eba13253
remote: error: See https://gh.io/lfs for more information.
remote: error: File node_modules/.cache/default-development/11.pack is 110.44 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.      
To https://github.com/DMitu/frizeriadd.git
 ! [remote rejected]   main -> main (pre-receive hook declined)
error: failed to push some refs to 'https://github.com/DMitu/frizeriadd.git'
i did everything i could find about this error and still can't solve this issue. Any advice would be great.

**Comment by soumyajit0** [comment]
This error message is indicating that you are trying to push a file named node_modules/.cache/default-development/11.pack to GitHub, but it is larger than the maximum file size limit allowed by GitHub, which is 100.00 MB.

To resolve this issue, you will need to remove the file from your repository and try pushing again. Here's what you can do:

1. Remove the file from your repository:
You can either delete the file locally or use the command git rm node_modules/.cache/default-development/11.pack to remove it from the repository.

2. Commit the changes:
After you have removed the file, commit the changes with a descriptive commit message using the command git commit -m "Removed file that exceeded GitHub's file size limit"

3. Push the changes to GitHub:
Finally, push the changes to GitHub using the command git push origin <branch-name>.

By removing the file that exceeds GitHub's file size limit, you should be able to push your changes to GitHub without encountering this error message.

**Reply by Kyl67899** [reply]
Have something similar to the error above but the file doesnt exist when I try to remove it.

remote: Resolving deltas: 100% (10469/10469), completed with 7 local objects.
remote: error: Trace: 9e8de94cd6b94ce6831349fa01d67e12b95b316f4b59ac0d1b3f251783dc001f
remote: error: See https://gh.io/lfs for more information.
remote: error: File node_modules/@next/swc-darwin-arm64/next-swc.darwin-arm64.node is 121.73 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.

**Reply by airtower-luna** [reply]
See my old response below: Removing the file in a new commit is not enough, you need to remove it entirely from the history.

**Comment by DMitu** [comment]
$ git rm --cached node_modules/.cache/default-development/11.pack
fatal: pathspec 'node_modules/.cache/default-development/11.pack' did not match any files
This pops whenever i try to delete it..

**Reply by soumyajit0** [reply]
Please make sure your total file size does not exceed 100 mb. After that you can try pushing your files to your repo. As I can see your repo : 'https://github.com/DMitu/frizeriadd.git' is currently empty, I am assuming you are using an external IDE to modify your codes. I expect that once you can make effective changed to your code (given it is less than 100 mb) you can push it without any issue.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @DMitu ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by airtower-luna** [comment]
Removing the large file in a new commit won't work, you have to remove it from the whole repository history. Otherwise it'll still be pushed (or rather fail to transfer) as part of the older commit that introduced it.

See https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository for ways to do that, the process is the same whether you want to remove a file from history because it's secret or too large. :wink:
